examples/crowdfunding/test/crowdfundingTest.js

```diff
@@ -233,11 +240,42 @@ describe('Crowdfunding Tests', function () {
         toAccountAddr: creator.address,
         amountMicroAlgos: 0,
         lsig: escrow,
-        payFlags: { closeRemainderTo: creator.address }
+        payFlags: { totalFee: 1000, closeRemainderTo: creator.address }
       }
     ];
+    const creatorBal = creator.balance(); // creator's balance before 'claim' tx
+    const escrowFunds = escrow.balance(); //  funds in escrow
+    // execute transaction
     await runtime.executeTx(txGroup, program, []);
-    // TODO- close account and tranfer funds to closeRemainderTo in runtime
+
+    syncAccounts();
+    assert.equal(escrow.balance(), 0); // escrow should be empty after claim
+    assert.equal(creator.balance(), creatorBal + escrowFunds - 2000); // funds transferred to creator from escrow
+
+    // after claiming, creator of the crowdfunding application should be able to delete the application
+    // NOTE: we don't need a txGroup here as escrow is already empty
+    const deleteTx = {
+      type: TransactionType.DeleteSSC,
+      sign: SignType.SecretKey,
+      fromAccount: creator.account,
+      appId: applicationId,
+      payFlags: { totalFee: 1000 },
+      appArgs: [],
+      accounts: [escrow.address] //  AppAccounts
+    };
+
+    const app = runtime.getApp(applicationId);
+    assert.isDefined(app); // verify app is present before delete tx
+
+    // execute delete tx
+    await runtime.executeTx(deleteTx, program, []);
+
+    // should throw error as app is deleted
```

Comment:
please add a assert here that funds are closed to creator account

Reply:
Ok, but just to confirm, we have already emptied the escrow before deleting the application so this assert will be same as lines 252-253